SimSim/Actions.swift

```diff
@@ -16,6 +16,7 @@ class Actions: NSObject
     @objc
     class func copy(toPasteboard sender: NSMenuItem)
     {
+        // This is the path to the object
         let path = sender.representedObject as! String
```

Comment:
Just a suggestion:
Why writing a comment if the variable-name could do all the necessary documentation?
E.g.: instead of path, name it pathToObject